comixed-webui/src/app/collections/pages/collection-detail/collection-detail.component.ts

```diff
@@ -116,7 +118,8 @@ export class CollectionDetailComponent implements OnInit, OnDestroy {
                   );
                 case CollectionType.SERIES:
                   return (
-                    (comicBook.series || '[UNKNOWN]') === this.collectionName
+                    (comicBook.series || '[UNKNOWN]') === this.collectionName &&
+                    comicBook.volume === this.volume
```

Comment:
if we don't have a volume (vnull), the route(http://localhost:4200/library/collections/series/Superman/volumes/null) leads to nothing (no comics in the page).